services/account-without-juggler/repositories/account/datasources/mysqlconn.ts

```diff
@@ -0,0 +1,201 @@
+const debug = require("debug")("account-without-juggler");
+const mysql = require("mysql");
+const db = require("mysql-promise")();
+import { CrudConnector } from "loopback-next/packages/repository/src/crud-connector";
+import { IError } from "@types/mysql";
+import {
+  Class,
+  DataSource,
+  Entity,
+  EntityData,
+  Filter,
+  ObjectType,
+  Options,
+  Where
+} from "loopback-next/packages/repository";
+
+export class MySqlConn implements CrudConnector {
+  private connection: any;
```

Comment:
Let's not prefix with I. You can create your own type or interface: See https://github.com/strongloop/loopback-next/blob/master/packages/core/src/internal-types.ts.  If we need to create a proxy to normalize the interfaces, so be it. cc @raymondfeng